alacritty/src/display/cursor.rs

```diff
@@ -90,3 +91,9 @@ fn hollow(x: f32, y: f32, width: f32, height: f32, thickness: f32, color: Rgb) -
         index: 0,
     }
 }
+
+/// For CursorShape::Block, drawing is done by changing 'bg' of cell,
+/// not drawing rect (see content.rs), so return default/empty value.
+fn block() -> CursorRects {
+    CursorRects::default()
+}
```

Comment:
How does this help?

Reply:
@chrisduerr , please check #7833 again, Kirill Chibisov explained it there.